ui/src/app/modules/login/login.component.ts

```diff
@@ -10,6 +10,7 @@ export class LoginComponent {
   private loginError :string
 
   constructor(private router: Router, private route: ActivatedRoute) {
+    localStorage.removeItem('id_token');
```

Comment:
This is only called when we render the login page (so the login route). If you head back to somewhere like `/app/applications` you'll be fine.

Do you think that we should update this to check for an expiration and then redirect/`removeItem` accordingly?